app/helpers/content_block_helper.rb

```diff
@@ -0,0 +1,20 @@
+module ContentBlockHelper
+  include Sufia::ContentBlockHelperBehavior
+
+  def displayable_content_block(content_block, **options)
+    return if content_block.value.blank?
+    content_tag :div, content_block.value, options
+  end
+
+  def editable_content_block(content_block)
+    capture do
+      concat form_for([sufia, content_block]) { |f|
+        concat f.label :value, content_block.name.titleize
+        concat f.text_area :value, class: "tinymce", rows: 20, cols: 120
+        concat f.submit 'Save', class: "btn btn-primary"
+      }
+      concat tinymce_assets
```

Comment:
It looks like this causes duplication of the tinymce assets:

![screen shot 2016-04-22 at 8 31 18 am](https://cloud.githubusercontent.com/assets/111218/14746469/9d5039c0-0864-11e6-9df6-cf1f4338a1db.png)

It looks like there's some ways to make it play nice with the asset pipeline that we ought to look at:

https://github.com/spohlenz/tinymce-rails#asset-compilation